pages/data-providers/data-provider.jsx

```diff
@@ -41,39 +47,37 @@ const useDataProviderController = () => {
 const getHelperMessage = ({ query, created, duplicated, error }) => {
   if (error) {
     return {
-      message:
-        'We could not find any OAI-PMH endpoint.' +
-        ' Please try it with another URL.',
-      variant: 'error',
+      message: <UnknownDataProviderHelperMessage />,
+      variant: UnknownDataProviderHelperMessage.variant,
     }
   }
 
   if (created) {
     return {
-      message: `We found OAI-PMH endpoint at ${created.oaiPmhEndpoint}`,
-      variant: 'success',
+      message: (
+        <NewDataProviderHelperMessage oaiPmhEndpoint={created.oaiPmhEndpoint} />
```

Comment:
I would love to have something like we had previously: 

```js
texts.messages.newDataProvider.render({ oaiPmhEndpoint: created.oaiPmhEndpoint })
```

We will need a custom class for it and a custom loader that adds this class to every `.mdx`.

However, I think these messages will not be ever edited by a content manager so we can put everything in a single file. Think about.

Reply:
I didn't find a simple way to have multiple components in a single file. We don't need to extract these messages. They can stay where they are. It's more like a proposal.

Reply:
I don't like JSX syntax in props, it does not look like the web should, in my opinion. However, if this works for you, I am fine with that.

Reply:
It can be changed to something like: 
```
 message: NewDataProviderHelperMessage ({ oaiPmhEndpoint: 'jojo' }),
```

or
```
 message: renderMessage({ oaiPmhEndpoint: 'jojo' }),
```


Nonetheless under the hood it's still a component. Let me know if you prefer this style. I don't mind any of it.

Reply:
Yes, I don't mind it's a component under the hood. I only want to hide this under some API that can be similar for the pure strings, markdown, MDX or any other tool we decide to use. Abstraction is a goal for this.

We may utilise the `.toString()` method enabling it to accept parameters or use a template class as before. I prefer a class instance with a method for this but a function you propose is fine too. I just have no idea how to do this now easily but if you are not scared of work, I will tell you details 🙂

If you don't want to overwork on this, the current solution if good as I said 👍